assets/src/application/ui/layout/entityList/EntityList.tsx

```diff
@@ -3,9 +3,8 @@ import { __ } from '@wordpress/i18n';
 import { Divider, Typography } from 'antd';
 
 import { EmptyState, ErrorIndicator, LoadingIndicator } from '@appDisplay/index';
-import EntityListFilterBar from '@appLayout/entityList/filterBar/EntityListFilterBar';
-import useEntityListFilterState from '@appLayout/entityList/filterBar/useEntityListFilterState';
-import { useEntityPagination } from '@appLayout/entityList/pagination';
+import { EntityListFilterStateManager } from './filterBar';
+import EntityListFilterBar from './withValidFilterState';
```

Comment:
should this reflect the actual file name?
`import withValidFilterState from './withValidFilterState';`

Reply:
May be, but it has to be PascalCase to be able to be used as JSX element.